.bazelversion

```diff
@@ -1 +1 @@
-2.1.0
+3.0.0
```

Comment:
This should be `3.0.1` correct? so that it matches the peer dep of the `bazel_toolchain`?

Reply:
There is no Bazel 3.0.1 unless I miss something here. The toolcbain repo is not really always in sync w/ Bazel based on how they released in the past.

Reply:
Sorry, you're right, I was under the impression they were in sync.